MAUI/Cartesian-Charts/neumorphicUI-column-chart.md

```diff
@@ -0,0 +1,245 @@
+            }
+
+            else
+            {
+                ApplyShadow(canvas, new RectF(dirtyRect.Left, dirtyRect.Top, -dirtyRect.Width, dirtyRect.Height), Offset, ShadowColor, Opacity, radius);
+                ApplyShadow(canvas, new RectF(dirtyRect.Left - 10, dirtyRect.Top, dirtyRect.Width + 10, -dirtyRect.Height), Offset, ShadowColor, Opacity, radius);
+                ApplyShadow(canvas, new RectF(dirtyRect.Right, dirtyRect.Top, dirtyRect.Width, dirtyRect.Height), LightOffSet, lightShadowColor, LightOpacity, radius);
+                ApplyShadow(canvas, new RectF(dirtyRect.Left, dirtyRect.Bottom, dirtyRect.Width, dirtyRect.Height), LightOffSet, lightShadowColor, LightOpacity, radius);
+            }
+        }
+        else
+        {
+
+            var paddingRect = new RectF() { Left = dirtyRect.Left + Padding, Top = dirtyRect.Top + Padding, Right = dirtyRect.Right - Padding, Bottom = dirtyRect.Bottom - Padding };
+            double radius = CornerRadius > paddingRect.Width / 2 ? paddingRect.Width / 2 : CornerRadius;
+            ApplyShadow(canvas, paddingRect, Offset, ShadowColor, Opacity, radius);
+            ApplyShadow(canvas, paddingRect, LightOffSet, lightShadowColor, LightOpacity, radius);
+        }
+    }
+
+{% endhighlight SfNeumorphismDrawer %}
+
+{% endtabs %}
+
+## Step 5: Implementing a SfNeumorphismView Class
+
+The `SfNeumorphismView` class extends `ContentView` and provides a container for displaying a neumorphic (soft shadow) effect. It includes a `Grid` and a `GraphicsView` for rendering the neumorphic shadow based on the provided `SfNeumorphismDrawer`. Additionally, it exposes properties and methods to manage the drawable content and handle property changes.
+
+{% tabs %}
+
+{% highlight SfNeumorphismView %}
+
+    public SfNeumorphismView()
+    {
+        Drawable = new SfNeumorphismDrawer();
+
+        grid = new Grid();
+        grid.Margin = new Thickness(0);
+
+        graphicsView = new GraphicsView();
+        graphicsView.Margin = new Thickness(0);
+        graphicsView.BackgroundColor = Colors.Transparent;
+        graphicsView.SetBinding(GraphicsView.DrawableProperty, new Binding() { Path = nameof(Drawable), Source = this });
+
+        grid.Children.Add(graphicsView);
+        base.Content = grid;
+    }
+    public void Invalidate()
+    {
+        graphicsView.Invalidate();
+    }
+
+{% endhighlight SfNeumorphismView %}
+
+{% endtabs %}
+
+## Step 6: Implementing the SfNeumorphismColumnSeries and SfNeumorphismColumnSegment Classes
+
+The `SfNeumorphismColumnSeries` class is a custom implementation of the `ColumnSeries` in the Syncfusion.Maui.Charts library. It introduces a neumorphic (soft shadow) effect to column series by associating a `SfNeumorphismDrawer` with each column segment.
+
+This class overrides the base method to create a custom `SfNeumorphismColumnSegment` instead of the default `ColumnSegment`. Associates the `Drawable` property with the newly created segment.
```

Comment:
@DhanarajRajendran 
Include code snippet

Reply:
Added the code snippet for both series and segment, below their contents.